app/jobs/reports/sp_issuer_user_counts_report.rb

```diff
@@ -13,17 +15,43 @@ def perform(_date)
           Db::Identity::SpUserCounts.with_issuer(issuer)
         end
 
+        reports = [
+          Reporting::EmailableReport.new(
+            title: 'Overview',
+            table: overview_table(issuer),
+          ),
+          Reporting::EmailableReport.new(
+            title: 'User counts',
+            table: user_table(user_counts),
+          ),
+        ]
+
         emails.each do |email|
-          ReportMailer.sp_issuer_user_counts_report(
-            name: REPORT_NAME,
-            email: email,
-            issuer: issuer,
-            total: user_counts['total'],
-            ial1_total: user_counts['ial1_total'],
-            ial2_total: user_counts['ial2_total'],
+          ReportMailer.tables_report(
+            email:,
+            subject:,
+            message:,
+            reports:,
+            attachment_format: :csv,
           ).deliver_now
         end
       end
     end
+
+    def overview_table(issuer)
+      [
+        ['Report Generated', Date.today.to_s], # rubocop:disable Rails/Date
```

Comment:
you can do `Time.zone.today` to get around this rubocop
```suggestion
        ['Report Generated', Time.zone.today.to_s],
```

Reply:
oh sure! i was just keeping it consistent with what you'd used in the idv report, as i figured there had probably been a reason for it! happy to just update it to `Time.zone.today` -- i will also update the other reports to make it consistent.

Reply:
LOL I think that was because those reports were loaded from the command line and didn't have Rails set, so `Time.zone` was nil. Inside this background job will always be Rails so it's fine

Reply:
ah, whoops